docs/reference/certificates.md

```diff
@@ -0,0 +1,121 @@
+| kube-etcd-healthcheck-client | etcd/healthcheck-client.key  | etcd/healthcheck-client.crt | etcdctl[2]     | --key                        | --cert                                    |
+
+## User configuration
+
+The administrator account and service accounts will need to be manually configured. 
+
+| filename                | credential name            | CN                             | O (in Subject) |
+|-------------------------|----------------------------|--------------------------------|----------------|
+| admin.conf              | default-admin              | kubernetes-admin               | system:masters |
+| kubelet.conf            | default-auth               | system:node:<hostname>         | system:nodes   |
+| controller-manager.conf | default-controller-manager | system:kube-controller-manager |                |
+| scheduler.conf          | default-manager            | system:kube-scheduler          |                |
+
+The first step is to generate an x509 cert/key pair with the given CN and O. Then, `kubectl` can be used to generate a config (one per line):
+
+```shell
+KUBECONFIG=<path> kubectl config set-cluster default-cluster --server=https://<host ip>:6443 --certificate-authority <path to kubernetes ca> --embed-certs
+KUBECONFIG=<path> kubectl config set-credentials <credential-name> --client-key <path-to-key>.pem --client-certificate <path-to-cert>.pem --embed-certs
+KUBECONFIG=<path> kubectl config set-context default-system --cluster default-cluster --user <credential-name>
+KUBECONFIG=<path> kubectl config use-context default-system
+```
+
+All these kubeconfigs should be copied to locations where they can be used. They should be provided with `--kubeconfig`.
+
+| name                       | path                    | command                 | comment                                                     |
+|----------------------------|-------------------------|-------------------------|-------------------------------------------------------------|
+| default-admin              | admin.conf              | kubectl                 | used by a user to administer the cluster                    |
+| default-auth               | kubelet.conf            | kubelet                 | one will be needed for every node in the cluster.           |
+| default-controller-manager | controller-manager.conf | kube-controller-manager | add to manifest in `manifests/kube-controller-manager.yaml` |
+| default-scheduler          | scheduler.conf          | kube-scheduler          | add to manifest in `manifests/kube-scheduler.yaml`          |
+
+[usage]: https://tools.ietf.org/html/rfc5280#section-4.2.1.3
+[kubeadm]: https://kubernetes.io/docs/reference/setup-tools/kubeadm/kubeadm/ 
+[proxy]: https://kubernetes.io/docs/tasks/access-application-cluster/access-cluster/#discovering-builtin-services
```

Comment:
The front-proxy CA/certs are used for https://kubernetes.io/docs/concepts/api-extension/apiserver-aggregation/